deployment/packer/template.js

```diff
@@ -62,6 +62,14 @@
             "instance_type": "t2.large",
             "ssh_username": "ubuntu",
             "ami_name": "mmw-worker-{{timestamp}}-{{user `version`}}",
+            "ami_block_device_mappings": [
+                {
+                    "device_name": "/dev/sdf",
+                    "snapshot_id": "snap-7bf80133",
```

Comment:
I think that we could copy this snapshot in order to rename it, and then ensure that it is tagged for the `Production` environment, so that we never delete it.